app/views/tags/show.haml

```diff
@@ -10,6 +10,8 @@
 
 .container-fluid#tags_show
   .row
+    #left-navbar-fixed-sub-background.col-lg-1.visible-lg
+    #left-navbar-fixed-background.col-md-offset-1.col-md-3.hidden-sm.hidden-xs
```

Comment:
Classes should be listed before IDs (.col-md-offset-1 should precede #left-navbar-fixed-background)